docs/how-to/download-new-upstream-version.rst

```diff
@@ -1,9 +1,139 @@
 ..  _download-new-upstream-version:
 
-===============================
-Download a new Upstream Version
-===============================
+Download a new :term:`Upstream` version
+=======================================
 
 .. caution::
 
     Work in progress!
+
+Once in a while you need to download a new :term:`Upstream` release or check if a newer
+:term:`Upstream` release exists.
+
+Most of the :term:`Source Packages <Source Package>` contain a ``watch`` file in the ``debian``
+folder. This is a configuration file for the :manpage:`uscan(1)` utility which allows you to
+automatically search :term:`HTTP` or :term:`FTP` sites or :manpage:`git(1)` repositories
+for newly available updates of the :term:`Upstream` project.
+
+.. note::
+    If the :term:`Source Package` does not contain a :file:`debian/watch` file, there may be
+    an explanation and instructions in the :file:`debain/README.source` or
+    :file:`debian/README.debian` file (if available) that tell you how to proceed.
+
+.. caution::
+    You should download :term:`Upstream` file(s) manually only if there is no automatic
```

Comment:
As a general suggestion, I'd say we should only define terms via the glossary the first time they appear in the page, the same way we would with an unexpanded acronym. Otherwise it's much more work than it should be to write an article.

Reply:
My thought was – if we reference the terms always throughout the article – readers which only read parts of an article have a much better time, because they don't have to search for the first appearance.

If you still think that this is not worth it, than I will only include the first appearance.

Reply:
I think it's generally fair to assume (at least, it's my working principle, for what it's worth) that someone who is new would be more likely to read the whole thing, while those who skip ahead to the bits they already know they want probably won't need the glossary definitions so much. There's definitely no harm in leaving them in if you want to, but since you've got a lot of writing ahead I wanted to share that perspective to save you potentially a lot of time and energy.

Reply:
I will continue with the "always reference" approach for now, because I actually use the search and replace feature of my editor; therefore it is not much more work, but I keep this advice in the back of my mind and look how it goes.